st2tests/tests/actions/render.py

```diff
@@ -0,0 +1,7 @@
+from st2common.runners.base_action import Action
```

Comment:
The tests pack I'm asking you to modify is located @ https://github.com/StackStorm/st2tests/tree/master/packs/tests.  It is a separate repo.  You're adding files in the wrong place.